gc/base/standard/ConcurrentGC.cpp

```diff
@@ -1963,6 +1964,8 @@ MM_ConcurrentGC::internalPreCollect(MM_EnvironmentBase *env, MM_MemorySubSpace *
 		env->_cycleState->_gcCode = MM_GCCode(gcCode);
 		env->_cycleState->_activeSubSpace = subSpace;
 		env->_cycleState->_collectionStatistics = &_collectionStatistics;
+		_extensions->globalGCStats.gcCount += 1;
```

Comment:
too late, we should do it in acquireExclusiveVMAccessAndSignalThreadsToActivateWriteBarrier (same just before that getUniqueGCCycleCount)